halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java

```diff
@@ -47,19 +47,19 @@
   String context;
   String cluster;
   String user;
-  @ValidForSpinnakerVersion(lowerBound = "1.5.0", message = "Spinnaker does not support configuring this behavior before that version.")
+  @ValidForSpinnakerVersion(lowerBound = "1.5.0", upperBound = "", tooLowMessage = "Spinnaker does not support configuring this behavior before that version.", tooHighMessage = "")
```

Comment:
I had assumed there was a way to keep parameters default/non-required -- do we need this `upperBound = ""` for each use of the annotation?

Reply:
I likewise assumed I could declare some `default` methods (ala `getUpperBound()`, `setUpperBound(String)`) on the interface but wasn't able to get the syntax into a format that intellij accepted.

Reply:
Ach, got it, @danielpeach pointed me towards how to declare defaults in annotation.